src/routes.js

```diff
@@ -16,14 +17,15 @@ const Routes = () => {
       <Route component={Header} />
       <Switch>
         <Route exact path="/" component={Landing} />
+        <Route exact path="/about" component={About} />
         <Route path="/preference" component={PreferenceList} />
         <Route path="/feed" component={AgendaFeedContainer} />
         <Route path="/signin" component={SignIn} />
         <Route path="/faq" component={Faq} />
         <Route path="/form" component={FormContainer} />
         <Route path="/topics" component={TopicsSelectorContainer} />
       </Switch>
-      <Route component={Footer} />
+      <Route exact path="/" component={Footer} />
```

Comment:
What does adding `exact path=“/“` here do?

Reply:
keeps the footer from showing up on other pages, thought it looked awkward

Reply:
Ah, gotcha. Cool, thank you! FYI for UI stuff like that in the future its a good idea to post a before / after screenshot.

Reply:
And thanks for the feedback on the landing page, definitely not a final design just a starting point because we had wanted to get this done at our meeting. I'll change the button colors and get Eric/Jeff to sign off!